Factories/StatusReporter/Factory.py

```diff
@@ -26,9 +26,13 @@ def __str__(self):
       return result
 
 class Factory(object):
-   def __init__(self):
-      self.state = []
-      self.lastPriceEvent = 0
+   def __init__(self, config):
+      self.readyState = {
```

Comment:
This isn't used anywhere, it overlaps with the function of `self.state[]`
See `async def onReadyEvent(self, dealer):` for the current behavior